### ISSUE TITLE
0.0.26 broke my slides - linked to slides md file placed into subdirectories...

### ISSUE THREAD

**Issue by dacr** [webpro/reveal-md#62]
```
slides/test/test.md
test.md
```

**OK** for test.md all resources are found
**NOK** for slides/test/test.md, all resources are missing.

I would say that within the generated html, some slashes are missing : 

```
<!doctype html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <title>Reveal.js</title>
        <link rel="stylesheet" href="css/reveal.css">
        <link rel="stylesheet" href="css/theme/black.css" id="theme">
        <!-- For syntax highlighting -->
        <link rel="stylesheet" href="lib/css/zenburn.css">
```

with `/css/reveal.css` and so one it should work...

**Comment by juanda99** [comment]
Any solution?
0.0.25 doesn't work either.
Thank you!

**Comment by juanda99** [comment]
@dacr you were right
#64 fixes the issue

**Comment by webpro** [comment]
I've made an attempt to fix this with v0.0.32 I've just released.
